library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -183,6 +183,6 @@ class ChuckerInterceptor @JvmOverloads constructor(
 
     companion object {
         private val UTF8 = Charset.forName("UTF-8")
-        private val CONTENT_ENCODING = "Content-Encoding"
+        private const val CONTENT_ENCODING = "Content-Encoding"
```

Comment:
This change is already merged with #190 
I would suggest to remove this change in your PR to avoid conflicts resolution.

Reply:
I've rebased on top of `develop`, should be fine now